extensions/resource-deployment/src/services/tools/azdataTool.ts

```diff
@@ -112,38 +112,46 @@ export class AzdataTool extends ToolBase {
 	private get azdataInstallAdditionalArgs(): string {
 		return vscode.workspace.getConfiguration(DeploymentConfigurationKey)[azdataPipInstallArgsKey];
 	}
+
+	protected dependenciesByOsType: Map<OsType, dependencyType[]> = new Map<OsType, dependencyType[]>([
+		[OsType.linux, [dependencyType.PythonAndPip3]],
+		[OsType.win32, [dependencyType.PythonAndPip3]],
+		[OsType.darwin, [dependencyType.PythonAndPip3]],
+		[OsType.others, [dependencyType.PythonAndPip3]]
+	]);
 }
 
 /*
 const linuxInstallationCommands = [
 	{
 		sudo: true,
-		comment: localize('resourceDeployment.Azdata.AptGetUpdate', "updating repository information ..."),
+		comment: localize('resourceDeployment.Azdata.AptGetUpdate', "updating repository information …"),
 		command: 'apt-get update'
 	},
 	{
 		sudo: true,
-		comment: localize('resourceDeployment.Azdata.AptGetPackages', "getting packages needed for azdata installation ..."),
+		comment: localize('resourceDeployment.Azdata.AptGetPackages', "getting packages needed for azdata installation …"),
 		command: 'apt-get install gnupg ca-certificates curl apt-transport-https lsb-release -y'
 	},
 	{
 		sudo: true,
-		comment: localize('resourceDeployment.Azdata.DownloadAndInstallingSigningKey', "downloading and installing the signing key for azdata ..."),
+		comment: localize('resourceDeployment.Azdata.DownloadAndInstallingSigningKey', "downloading and installing the signing key for azdata …"),
 		command: 'wget -qO- https://packages.microsoft.com/keys/microsoft.asc | apt-key add -'
 	},
+	// Double check with Sean Wells on the current url for azdata corresponding to https://packages.microsoft.com/config/ubuntu/16.04/mssql-server-preview.list)
 	{
```

Comment:
Why is this here? Tasks should be tracked somewhere else, not in code

Reply:
will remove the comment

Reply:
done